docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md

```diff
@@ -73,13 +77,17 @@ Alternatively, you can also do the same with a specific release by navigating to
 
   <img src="./media/SetupEnvironment/downloadZipRelease.png" alt="Download zip" height="150">
 
+To update your copy with the latest changes in CARML, you would need to download, for example, the latest CARML release the same way you did during your initial setup, copy its folder content to your repository's location, and upload the changes.
```

Comment:
Shouldn't this go in the "Fetch latest changes" section?

Reply:
Yes-ish. Apparently it wasn't obvious enough - so I would suggest to have this here with a link to the 'fetch changes' section for 'further information'.

Reply:
Update: Added it below.